generators/mailer/templates/mysql/passportLocal/src/modules/v1/users/users.service.ts

```diff
@@ -19,11 +19,11 @@ export default class UsersService {
     });
   }
 
-  public getByEmail(email: string): Promise<UserEntity | null> {
+  public getByEmail(email: string): Promise<UserEntity | undefined> {
     return this.usersRepository.getByEmail(email);
   }
 
-  public getById(id: number): Promise<UserEntity | null> {
+  public getById(id: number): Promise<UserEntity | undefined> {
     return this.usersRepository.getById(id);
```

Comment:
Promise<UserEntity | void> is more correct logic? 
https://stackoverflow.com/questions/58885485/why-does-typescript-have-both-void-and-undefined